src/players/YouTube.js

```diff
@@ -8,9 +8,19 @@ const SDK_URL = '//www.youtube.com/iframe_api'
 const SDK_GLOBAL = 'YT'
 const MATCH_URL = /^(?:https?:\/\/)?(?:www\.)?(?:youtu\.be\/|youtube\.com\/(?:embed\/|v\/|watch\?v=|watch\?.+&v=))((\w|-){11})(?:\S+)?$/
 const PLAYER_ID = 'youtube-player'
+const DEFAULT_PLAYER_VARS = {
+  autoplay: 1,
+  controls: 0,
+  showinfo: 0
+}
 
 export default class YouTube extends Base {
   static propTypes = propTypes
+  static defaultProps = {
+    youtube: {
+      playerVars: DEFAULT_PLAYER_VARS
```

Comment:
Maybe just inline the defaults here instead of using a `const`? To be consistent with the Vimeo player